cpp/include/cudf/transform.hpp

```diff
@@ -53,6 +53,12 @@ std::unique_ptr<column> transform(
   bool is_ptx,
   rmm::mr::device_memory_resource* mr = rmm::mr::get_current_device_resource());
 
+std::unique_ptr<column> generalized_masked_op(
+  table_view data_view,
```

Comment:
Typically we pass in `table_view const&` as copying it may involve recursively copying its children column_view which is more expensive.

Reply:
You may need to be modified to use `table_view const&` (not just this, but in other places too).